srcpkgs/radicale/files/radicale/run

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
 exec 2>&1
-exec chpst -u radicale radicale -f -C /etc/radicale/config
+exec chpst -u radicale radicale -C /etc/radicale/config
```

Comment:
If you have a compelling reason to use the `_radicale` user (see my objection to that), the user in this line would be wrong.

Reply:
Now its correct. :)